include/Core/Terms/Modules/DeleteTermsByNameModule.php

```diff
@@ -43,7 +43,9 @@ public function render() {
 	}
 
 	public function filter_js_array( $js_array ) {
-		$js_array['validators'][ $this->action ] = 'noValidation';
+		$js_array['validators'][ $this->action ] = 'validateTermName';
+		$js_array['error_msg'][ $this->action ]  = 'enterTermName';
+		$js_array['msg']['enterTermName']        = __( 'Please enter valid Term Name', 'bulk-delete' );
```

Comment:
Let's change this message to `Please enter the term name that should be deleted`

Reply:
Fixed.